Bottom: f5bce0d495ca8b68a6b3963f07ed85a2266cec9d
Top:    c87d40f9917c21534131c2f4ac21976842d261b2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 15:03:27 +0100

doc: Remove execute permissions on Makefile.am and gs/Makefile.am


---

diff --git a/doc/Makefile.am b/doc/Makefile.am
old mode 100755
new mode 100644
diff --git a/doc/gs/Makefile.am b/doc/gs/Makefile.am
old mode 100755
new mode 100644
